drivers/flash/Kconfig.at45

```diff
@@ -23,7 +24,7 @@ if SPI_FLASH_AT45
 
 config SPI_FLASH_AT45_USE_READ_MODIFY_WRITE
 	bool "Use Read-Modify-Write command in flash_write()"
-	default y
+	default n
```

Comment:
Why has this changed?

Reply:
This was changed because it could result in faster wear of these devices. I can change it back and threat these devices as devices without erase when this config is selected.

Reply:
I think that would make sense to conditionalize the erase based on this config. However this does not seem to be related to the intent behind this PR. So probably make this change a separate commit.